### PR TITLE
added sidebar permissions to manifest and added PowerShell build script

### PR DESCRIPTION
**Pull request by vaimalaviya1233**
I looked through the Firefox docs and added side bar implementation to Firefox manifest however i don't use Firefox, so someone needs to test if it works, but i tested the chrome version on Microsoft edge and it works.

Thanks.

### CODE REVIEW

File: src/pages/Background/index.ts [hunk 1]
```diff
@@ -296,10 +297,15 @@ browser.omnibox.onInputEntered.addListener(async (content: string, disposition:
...
 
+  configuration = await getConfig();
+
   const isUrl = /^http(s)?:\/\//.test(content);
+
+
+// This part was taken https://github.com/sissbruecker/linkding-extension/blob/master/src/background.js Thanks to @sissbruecker
   const url = isUrl
     ? content
-    : `lk`;
```
[the remark this answers is not in the document]
Comment (vaimalaviya1233) [reply src/pages/Background/index.ts:302]: fix for #83

**Comment by crkos** [comment]
Thank you for the hard work!, Can you make this PR to dev instead so it can be tested then merged to main in case evertything is right

**Comment by vaimalaviya1233** [comment]
> Thank you for the hard work!, Can you make this PR to dev instead so it can be tested then merged to main in case everything is right

right now not sure because i might have to learn about extension framework a bit since i don't know about framwork, that's why and i made this pr but later i also tried to integrate latest patch which was done to dev branch which was related to settings to show options which was #75 but it broke the extension in my local test so i'm not sure which branch to start with. because dev branch is 30 commit behind main branch(idk how branches and merges go) but if i start with dev branch then when devs go to merge the dev and main to create release shouldn't it cause some problems bc of mismatched code. or should i just ignore about that and just make pr to dev?
